### ISSUE TITLE
Is EasyIB still working?

### ISSUE THREAD

**Issue by jongsuk-park** [ashpipe/EasyIB#12]
I am using a PAPER account for the test.

Some of requests work, but some of them do not work. 
Do you have any idea how to fix them?

① Get live orders

>>> Response:
{'error': 'Please query /accounts first'}

② Get auth status, Get ping server
→ This one works, but why I am not still authenticated?

>>> Response:
{'authenticated': False, 
'competing': False,
'connected': False,
'MAC': 'F4:03:43:DA:99:00'}

{'session': '8ce8747753be3a9f1bebe610b3b743 c6',
'ssoExpires': 0,
'collission': False,
'userId': 102651152,
'hmds': {'error': 'no bridge'},
'iserver': {'authStatus': {'authenticated': False, 
'competing': False, 
'connected': False, 
'MAC': 'F4:03:43:DA:99:00'}}}

③ Switch account
>>> Response:
{'error': 'Init session first', 'statusCode': 400}

④ Get Bars (market data)

>>> Response:
{'error': 'Init session first', 'statusCode': 400}

**Comment by ashpipe** [comment]
It seems your account is not authenticated. Are you sure you logged in at https://localhost:5000 in a web browser?

**Comment by jongsuk-park** [comment]
@ashpipe 

Yeah, I logged in by typing my paper account username and password, and I saw
"Client login succeeds".

However, the login page did not ask me to enter security code by using my phone.

Is that the reason why I do not get authenticated?   Have you ever tried a paper account with EasyIB?

![image](https://github.com/ashpipe/EasyIB/assets/95947138/58baa800-7d55-443b-a3fd-a70fdf10ee52)

![image](https://github.com/ashpipe/EasyIB/assets/95947138/e18e2172-b9e6-43ef-8162-1a2a9dcc2300)
